crates/rune-modules/src/lib.rs

```diff
@@ -74,24 +74,9 @@
 //! * `process` for the [process module][process]
 //! * `rand` for the [rand module][rand]
 //! * `signal` for the [signal module][signal]
-//! * `test` for the [test module][test]
+//! * `test` for the [test module][module@test]
 //! * `time` for the [time module][time]
 //! * `toml` for the [toml module][toml]
-//!
-//! [core]: https://docs.rs/rune-modules/0/rune_modules/core/
-//! [experiments]: https://docs.rs/rune-modules/0/rune_modules/experiments/
-//! [fmt]: https://docs.rs/rune-modules/0/rune_modules/fmt/
-//! [fs]: https://docs.rs/rune-modules/0/rune_modules/fs/
-//! [http]: https://docs.rs/rune-modules/0/rune_modules/http/
-//! [io]: https://docs.rs/rune-modules/0/rune_modules/io/
-//! [json]: https://docs.rs/rune-modules/0/rune_modules/json/
-//! [macros]: https://docs.rs/rune-modules/0/rune_modules/macros/
-//! [process]: https://docs.rs/rune-modules/0/rune_modules/process/
-//! [rand]: https://docs.rs/rune-modules/0/rune_modules/rand/
-//! [signal]: https://docs.rs/rune-modules/0/rune_modules/signal/
-//! [test]: https://docs.rs/rune-modules/0/rune_modules/test/
-//! [time]: https://docs.rs/rune-modules/0/rune_modules/time/
-//! [toml]: https://docs.rs/rune-modules/0/rune_modules/toml/
```

Comment:
These are needed, because they're generated [as part of the README](https://github.com/rune-rs/rune/tree/main/crates/rune-modules) through `cargo-readme`.

Reply:
Restored and linked to livioribeiro/cargo-readme#55 to document this fact and track any future support cargo-readme might have for intra-doc links.